src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +50,36 @@
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService = (MovieSessionService) injector
+                .getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        AuthenticationService authenticationService = (AuthenticationService) injector
+                .getInstance(AuthenticationService.class);
+        try {
+            authenticationService.register("bob@mail.com", "password");
+        } catch (RegistrationException e) {
+            throw new RuntimeException("Can not register user", e);
+        }
+        try {
+            authenticationService.login("bob@mail.com", "password");
+        } catch (AuthenticationException e) {
+            throw new RuntimeException("Login unsuccessful", e);
+        }
+        try {
+            authenticationService.register("alice@mail.com", "qwerty");
```

Comment:
Registering a second user ('alice@mail.com') might not be necessary for demonstrating the functionality unless it's required by the task. Consider removing this if it's redundant.